### ISSUE TITLE
nglview on google colab

### ISSUE THREAD

**Issue by andresilvapimentel** [googlecolab/colabtools#2272]
I would like to suggest the project nglview on google colab.
It would be great if the nglview package could be used on google colab because it will beneficit many people working with chemical compounds and proteins.
Thanks!!!

**Comment by blois** [comment]
Thanks for the report. This is intended to work when support for third-party widgets is enabled but it looks like there's still a remaining issue there. I've opened https://github.com/googlecolab/colab-cdn-widget-manager/issues/13 to track further.

**Comment by blois** [comment]
This should now be working: https://colab.research.google.com/gist/blois/5ac0123ff66583f2578858fce7fdaf0b/nglview.ipynb

**Comment by Dan-Burns** [comment]
Did it break? 

I got the following error when I tried to run the linked notebook:

error: subprocess-exited-with-error
  
  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> See above for output.
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
  Preparing metadata (setup.py) ... error
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.
